### ISSUE TITLE
hologram disappears when i change IP or port

### ISSUE THREAD

**Issue by TienIC** [Creeperface01/Holograms#12]
It's all gone and only close enough to see a line. That is when I connect the Nemisys server, then I turn off the Nemisys server and connect normally to Nukkit. Then the hologram had completely disappeared. 1 more time, the hologram disappears when i change the server ip. For example, my server ip is 34.25.xxxx.34 switch to ip 35.26.xxxx.35 the dimensional image will disappear.But when I go back to the old ip, nod is back to normal.

**Comment by Creeperface01** [comment]
Nemysis is probably no longer supported. I know it worked in the past but there were certain issues that probably came out again. Use waterdog for example.

**Comment by TienIC** [comment]
No, Nemisys helped me to find this error. I tried running Nukkit without Nemisys, also error if I change IP

**Comment by Creeperface01** [comment]
Well then I don't understand what the issue is. Server IP has nothing to do with holograms or even the server itself. Or what do you mean by "switch" of ip address

**Comment by TienIC** [comment]
I mean: the hologram will disappear when I change the IP. Here is the photo
![Screenshot_20211109-034626_Minecraft](https://user-images.githubusercontent.com/43937423/140890811-d76836dc-3f5c-4efa-9073-b7725bbb60c5.png)

**Comment by BeanGTA** [comment]
![Uploading Screenshot_20211109-034622_Minecraft.png…]()

**Comment by BeanGTA** [comment]
![Uploading Screenshot_20211107-041406_Minecraft.png…]()

**Comment by TienIC** [comment]
yes my photo sent error. I will message you on discord

**Comment by TienIC** [comment]
Standing, you will see the text, but standing far away, it will disappear and nothing will be seen.

**Comment by Creeperface01** [comment]
what do you mean by changing IP? Transferring to another server and back? And how? transfer packet? or some kind of proxy?

**Comment by TienIC** [comment]
1: I got an error by changing the Port
2: I get error when I change IP from another server in VPS
3: I got an error when backing up the server file, and pasting a nukkit server file in the new VPS

My friends in the group, all got the same error as above. And this error has been since version 4.0 was released. One reviewer told you the holograms naturally disappeared. I'm a person who doesn't like the cause of the error... I did a reset, the plugin file is only a hologram. I researched for 4 days straight, and finally I tried to paste the server file with the old IP. So it went back to normal, nothing happened LOL...But I try to copy the server file again to another Linux server, it disappears. Only standing close and looking at it has 1 line. I surrender...
